README.md

```diff
@@ -84,6 +84,7 @@ jobs:
 | `github-token` | ❌        | `github.token` | The GitHub token used to call the GitHub API    |
 | `timeout`      | ❌        | `30000`        | timeout before we stop trying (in milliseconds) |
 | `delay`        | ❌        | `5000`         | delay between status checks (in milliseconds)   |
```

Comment:
this needs to go in `docs/readme.md` plz 

(sorry this my templated appraoch to keep readme.md in sync)

Reply:
done, hope it's ok